### ISSUE TITLE
Renaming Helm chart to `tofu-controller`

### ISSUE THREAD

**Issue by chanwit** [flux-iac/tofu-controller#1252]
Please let me know if it's going to break your current setup.

**Comment by Sierra1011** [comment]
@chanwit Apologies for nagging, it's not meant that way; I saw that #1264 merged last week but the Action to release the updated HelmChart versions/name hasn't run. Is there a specific hold due to external events (_cough xz_) or is this just imminently in the pipeline? I'm keen to update :grin:

**Comment by ilithanos** [comment]
@Sierra1011 the helm charts follow the tofu-controller versioning tightly, so it will be released together with the next rc version.

There are a few more PR's that I would like to have merged into that rc version before releasing, but it will be done soon ;)

**Comment by akselleirv** [comment]
@ilithanos do you have any more changes before we can create a new release?
